<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefixSettings</class>
 <widget class="QDialog" name="PrefixSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prefix Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../q4wine.qrc">
    <normaloff>:/data/q4wine.png</normaloff>:/data/q4wine.png</iconset>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_7" native="true">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Midlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Midlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Midlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout">
      <property name="leftMargin">
       <number>18</number>
      </property>
      <item>
       <widget class="QLabel" name="lblCaption">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Prefix Settings</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblLogo">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../q4wine.qrc">:/data/exec.png</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout">
      <item>
       <widget class="QTabWidget" name="twbGeneral">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>General</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="margin">
           <number>3</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Prefix Settings</string>
            </property>
            <layout class="QVBoxLayout">
             <property name="margin">
              <number>3</number>
             </property>
             <item>
              <widget class="QWidget" name="widget_16" native="true">
               <layout class="QHBoxLayout">
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_15">
                  <property name="minimumSize">
                   <size>
                    <width>140</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>140</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Prefix path:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtPrefixPath"/>
                </item>
                <item>
                 <widget class="QPushButton" name="cmdGetPrefixPath">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>16</width>
                    <height>16</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_15" native="true">
               <layout class="QHBoxLayout">
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_14">
                  <property name="minimumSize">
                   <size>
                    <width>140</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>140</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Name:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtPrefixName"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_19" native="true">
               <layout class="QHBoxLayout" name="_2">
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_18">
                  <property name="minimumSize">
                   <size>
                    <width>140</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>140</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Architecture:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboArchList">
                  <item>
                   <property name="text">
                    <string>Default</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">win32</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">win64</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_7">
            <property name="title">
             <string>Quick Mount</string>
            </property>
            <layout class="QVBoxLayout">
             <property name="margin">
              <number>3</number>
             </property>
             <item>
              <widget class="QWidget" name="widget_18" native="true">
               <layout class="QHBoxLayout">
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_17">
                  <property name="minimumSize">
                   <size>
                    <width>140</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>140</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Mount point:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtMountPoint"/>
                </item>
                <item>
                 <widget class="QPushButton" name="cmdGetMountPoint">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>16</width>
                    <height>16</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_3" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_19">
                  <property name="minimumSize">
                   <size>
                    <width>140</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>140</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Device:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboWinDrive">
                  <item>
                   <property name="text">
                    <string notr="true">D:</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">E:</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">F:</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">G:</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">H:</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">I:</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">J:</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">K:</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">L:</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">M:</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">N:</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">O:</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">P:</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">Q:</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">R:</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">S:</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">T:</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">U:</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">V:</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">W:</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">X:</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">Y:</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>84</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Wine settings</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="margin">
           <number>3</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Wine settings</string>
            </property>
            <layout class="QVBoxLayout">
             <property name="margin">
              <number>3</number>
             </property>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>If you wish to use different Wine versions, you may wish to set custom Wine paths here.
Note: leave it blank for default.</string>
               </property>
               <property name="scaledContents">
                <bool>false</bool>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_2" native="true">
               <layout class="QHBoxLayout">
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="minimumSize">
                   <size>
                    <width>140</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>140</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Bin:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtWineBin"/>
                </item>
                <item>
                 <widget class="QPushButton" name="cmdGetWineBin">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>16</width>
                    <height>16</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_4" native="true">
               <layout class="QHBoxLayout">
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="minimumSize">
                   <size>
                    <width>140</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>140</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Server:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtWineServerBin"/>
                </item>
                <item>
                 <widget class="QPushButton" name="cmdGetWineServerBin">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>16</width>
                    <height>16</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_10" native="true">
               <layout class="QHBoxLayout">
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="minimumSize">
                   <size>
                    <width>140</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>140</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Loader:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtWineLoaderBin"/>
                </item>
                <item>
                 <widget class="QPushButton" name="cmdGetWineLoaderBin">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>16</width>
                    <height>16</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_11" native="true">
               <layout class="QHBoxLayout">
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="minimumSize">
                   <size>
                    <width>140</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>140</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Libs:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtWineLibs"/>
                </item>
                <item>
                 <widget class="QPushButton" name="cmdGetWineLibs">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>16</width>
                    <height>16</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>74</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QPushButton" name="cmdHelp">
          <property name="text">
           <string>Help</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="cmdCancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cmdOk">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../q4wine.qrc"/>
 </resources>
 <connections/>
</ui>
